mozilla-release/mobile/android/tests/cliqz/java/com/cliqz/mobile/test/driver/ElementData.java

```diff
@@ -0,0 +1,33 @@
+package com.cliqz.mobile.test.driver;
+
+import android.support.test.uiautomator.UiObject2;
+import android.util.Log;
+
+public class ElementData {
+
+    private UiObject2 element;
+    private boolean exists, checked, enabled, focused, selected;
```

Comment:
One declaration per line would be more readable

Reply:
done.